evenodd.c

```diff
@@ -0,0 +1,16 @@
+/* Can Parlar - Checks if the input is odd or even */
+
+#include <stdio.h>
+
+int main() {
+	int userInput;
+	int remainder;
+	printf("Enter a number:");
+	scanf("%d", &userInput);
+remainder = userInput % 2;
+if (userInput == 0 || remainder == 0) {
```

Comment:
you actually don't need the userInput == 0 because 0%2 will still have a remainder of 0, so that part is redundant.